modules/Home/controller.php

```diff
@@ -89,19 +89,26 @@ public function action_getValidationRules(){
 
                 $fielddef = $bean->field_defs[$_REQUEST['field']];
 
-                if(!$fielddef['required']){
+                if(!isset($fielddef['required']) || !$fielddef['required']){
                     $fielddef['required'] = false;
                 }
 
-                if($fielddef['name'] == "email1" || $fielddef['email2']){
+                if ($fielddef['name'] == "email1" || (isset($fielddef['email2']) && $fielddef['email2'])) {
```

Comment:
It's an other issue because its change the behavior of home controller. How should it works?